### ISSUE TITLE
I am asking you to bring back Linux support.

### ISSUE THREAD

**Issue by Pythone** [Rydion/Snapp#53]
I use Linux, and more people are using it. I think that by adding Linux support back. You will have more OSes to cover. Also, Linux is the OS that most good developers use.I think native Linux support is vital.

**Comment by Blockguy24** [comment]
I, too would like to see Linux support re-implemented. A file format such as AppImage can be used to contain the whole application in a single file.

**Comment by opencoca** [comment]
Yes, with the growing number of schools using Raspberry Pi systems and Chrome Books, students have never used Linux more. Now is not the time to dump Linux builds. 

Where's the last release of Snapp that works for Linux?

**Comment by Pythone** [comment]
I do not see it. The only release on the releases page was v6(which dropped support).
